source/postprocess/parallel_statistics.cc

```diff
@@ -0,0 +1,88 @@
+#include <aspect/simulator.h>
+
+#include <boost/archive/text_oarchive.hpp>
+#include <boost/archive/text_iarchive.hpp>
+
+
+namespace aspect
+{
+    namespace Postprocess
+    {
+        template <int dim>
+        std::pair<std::string,std::string>
+        ParallelStatistics<dim>::execute (TableHandler &statistics)
+        {
+            // memory consumption:
+            const double mb = 1024*1024; //convert from bytes into mb
+            statistics.add_value ("Triangulation memory consumption (MB) ", this->get_triangulation().memory_consumption()/mb);
+            statistics.add_value ("p4est memory consumption (MB) ", this->get_triangulation().memory_consumption_p4est()/mb);
+            
+//            << "* DoFHandler " << dof_handler.memory_consumption()/mb <<std::endl
+//            << "* ConstraintMatrix " << constraints.memory_consumption()/mb << std::endl
+//            << "* current_constraints " << current_constraints.memory_consumption()/mb << std::endl
+//            << "* Matrix " << system_matrix.memory_consumption()/mb << std::endl
+//            << "* 5 Vectors " << 5*solution.memory_consumption()/mb << std::endl
+//            << "* preconditioner " << (system_preconditioner_matrix.memory_consumption()
+//                                       //                                     + Amg_preconditioner->memory_consumption()
+//                                       /*+Mp_preconditioner->memory_consumption()
+//                                        +T_preconditioner->memory_consumption()*/)/mb
+//            << std::endl
+//            << "  - matrix " << system_preconditioner_matrix.memory_consumption()/mb << std::endl
+//            //          << "  - prec vel " << Amg_preconditioner->memory_consumption()/mb << std::endl
+//            << "  - prec mass " << 0/*Mp_preconditioner->memory_consumption()/mb*/ << std::endl
+//            << "  - prec T " << 0/*T_preconditioner->memory_consumption()/mb*/ << std::endl
+//            << std::endl;
+            
+            std::ostringstream output;
+            output << this->get_triangulation().memory_consumption()/mb << ", "
+            << this->get_triangulation().memory_consumption_p4est()/mb;
+            
+            return std::pair<std::string, std::string> ("Memory consumption tria/p4est:",
+                                                        output.str());
+
+        }
+    }
+}
+
+
+// explicit instantiations
+namespace aspect
+{
+    namespace Postprocess
+    {
+        ASPECT_REGISTER_POSTPROCESSOR(ParallelStatistics,
+                                      "parallel statistics",
+                                      "A postprocessor that computes some statistics about "
+                                      "the parallel solution process. "
+                                      "In particular, it computes minimal, average and maximal "
+                                      "values of particles per cell in the global domain.")
```

Comment:
values of -> number of

But this doesn't seem to match what you do above. Is this a copy-paste issue?